batches.go

```diff
@@ -0,0 +1,120 @@
+package packngo
+
+import (
+	"fmt"
+)
+
+const batchBasePath = "/batches"
+
+// BatchService interface defines available batch methods
+type BatchService interface {
+	Get(batchID string, listOpt *ListOptions) (*Batch, *Response, error)
+	List(ProjectID string, listOpt *ListOptions) ([]Batch, *Response, error)
+	Create(projectID string, batches *InstanceBatchCreateRequest) ([]Batch, *Response, error)
+	Delete(string, bool) (*Response, error)
+}
+
+// Batch type
+type Batch struct {
+	ID                     string     `json:"id"`
+	State                  string     `json:"state,omitempty"`
+	Quantity               int32      `json:"quantity,omitempty"`
+	CreatedAt              *Timestamp `json:"created_at,omitempty"`
+	Href                   string     `json:"href,omitempty"`
+	Project                Href       `json:"project,omitempty"`
+	Instances              Href       `json:"instances,omitempty"`
```

Comment:
I'm a little confused by the use of `Instances` and `Devices` here - what is the difference meant to be? I don't believe the API returns an `instances` field (maybe it did at one point?)

Reply:
From the calls I was making earlier today it seems to not be there anymore, maybe it's been renamed to `devices`

Reply:
I'm also having some trouble serializing to `[]Device` as well, I can reach out via slack